resources/model/OpenStudio.idd

```diff
@@ -3252,6 +3252,135 @@ OS:MaterialProperty:MoisturePenetrationDepth:Settings,
        \note resistance to water vapor diffusion in stagnant air
        \note (see Water Vapor Diffusion Resistance Factor above).
 
+OS:MaterialProperty:PhaseChange,
+       \memo Additional properties for temperature dependent thermal conductivity
+       \memo and enthalpy for Phase Change Materials (PCM)
+       \memo HeatBalanceAlgorithm = CondFD(ConductionFiniteDifference) solution algorithm only.
+       \memo Constructions with this should use the detailed CondFD process.
+       \memo Has no effect with other HeatBalanceAlgorithm solution algorithms
+       \extensible:2
+       \max-fields 35
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field Material Name
+       \required-field
+       \type object-list
+       \object-list MaterialNames
```

Comment:
I think we should be clearer than E+. Make an additional `\reference MaterialWithPropertyNames` (or something better/clearer) on the StandardOpaque/MassLessMaterial ones and use that here in object-list instead of the generic MaterialNames.

Reply:
[e9677bc](https://github.com/NREL/OpenStudio/pull/4610/commits/e9677bc77b2f381653dafcbae3c874197b496eb2)